### PR TITLE
Fix test_suite location.

### PR DESCRIPTION
**Pull request by ExaneServerTeam**
Makes ``make test`` & coverage work again

### CODE REVIEW

**Comment by ExaneServerTeam** [comment]
Travis has to be fixed first. See #207

**Comment by ExaneServerTeam** [comment]
In wait for #207 to be merged.

**Comment by ExaneServerTeam** [comment]
@yuewko fix current make test issue. Would be a nice to merge ;)

`
======================================================================
ERROR: tests (unittest.loader._FailedTest)
----------------------------------------------------------------------
ImportError: Failed to import test module: tests
Traceback (most recent call last):
  File "/opt/rh/rh-python36/root/usr/lib64/python3.6/unittest/loader.py", line 153, in loadTestsFromName
    module = __import__(module_name)
ModuleNotFoundError: No module named 'infoblox_client.tests'


----------------------------------------------------------------------

`

File: setup.py [hunk 1]
```diff
@@ -50,6 +50,6 @@
         'Programming Language :: Python :: 3.4',
         'Programming Language :: Python :: 3.5',
     ],
-    test_suite='infoblox_client.tests',
+    test_suite='tests',
```
Comment (yuewko) [inline setup.py]: Could you please explain why this change is suddenly necessary? This line hasn't been changed for 3 years.

(Please bear with me since I haven't been following this project closely)

Comment (ExaneServerTeam) [reply setup.py]: Hello @yuewko ,

I imagine a story like this. A long time ago tests were performed with make test & the test suite was under infoblox_dir module directory.

The coverage went up, and tests location changed to roodir. See 4953dfb50691c8565c72545f97af58b42fb2acd9.

As test coverage is now part of the tox plan ( [See tox.ini](https://github.com/infobloxopen/infoblox-client/blob/master/tox.ini) ) with a direct nosetest call, this config was just forgotten along the way

Comment (yuewko) [reply setup.py]: Umm.. unless I'm missing, I don't think this explains it.

As far as I can tell, neither `.travis.yaml` nor `tox.ini` have been changed for at least 2 years - except adding `py36` around 8 months back. BTW, I believe that the `nosetest` call you mentioned was there 3 years ago.

Are you saying that the CI have been failing for 2-3 years? (which is unlikely, I believe)

Thanks for following up!

Comment (ExaneServerTeam) [reply setup.py]: @yuewko I think it's brocken for a while ...

1. It only fails when you run 'make test' or `python setup.py test` (give it a shot ;)) from the source tree.
Extract from Makefile
```
test:
    python setup.py test
```

2.  `make coverage` also runs through the `setup.py test` command
Extract of Makefile:
```
coverage:
    coverage run --source infoblox_client setup.py test
    coverage report -m
    coverage html
    $(BROWSER) `htmlcov/index.html
```


3. Travis & tox follow an other command path for tests
-> They never use 'make test' or `setup.py test` command
-> It runs `nosetest`
Extract from `tox.ini`
```
commands =
    mkdir -p {[setup]results}
    nosetests \
    ┊ --with-xunit \
    ┊ --xunit-file {[setup]results}/nose.xml \
    ┊ --with-coverage \
    ┊ --cover-erase \
    ┊ --cover-tests \
    ┊ --cover-branches \
    ┊ --cover-xml \
    ┊ --cover-xml-file {[setup]results}/coverage.xml \
    ┊ --cover-html \
    ┊ --cover-html-dir {[setup]results}/coverage \
    ┊ --cover-package infoblox_client

````

Comment (yuewko) [reply setup.py]: OK @ExaneServerTeam , thanks again for your explanation.

LGTM.
